components/webdriver_server/elements.rs

```diff
@@ -137,6 +133,7 @@ impl Handler {
                     .collect::<WebDriverResult<Vec<String>>>()?;
                 format!("{{{}}}", elems.join(", "))
             },
+            _ => serde_json::to_string(v).unwrap(),
```

Comment:
If this fails, maybe you can return a WebDriver error instead of panicking here?

```suggestion
            _ => serde_json::to_string(v).unwrap(),
```